src/main/kotlin/xyz/bluspring/kilt/loader/KiltLoader.kt

```diff
@@ -363,15 +383,17 @@ class KiltLoader {
         val thrownExceptions = mutableMapOf<String, Exception>()
         DeltaTimeProfiler.push(modFile.nameWithoutExtension)
 
-        Kilt.logger.debug("Scanning jar file ${modFile.name} for Forge mod metadata.")
+        Kilt.logger.debug("Scanning jar file {} for Forge mod metadata.", modFile.name)
```

Comment:
Please avoid using this style of string formatting, as it makes it more difficult to read.